modules/sor/balancer-sor.integration.test.ts

```diff
@@ -423,6 +438,420 @@ describe('Balancer SOR Integration Tests', () => {
         });
     });
 
+    describe('Pools Path with hooks -', async () => {
```

Comment:
Next step is to move some of these tests to a different file. It's starting to get a bit hard to follow this many changes in a single file 😅 
(should be ok to do that in a following PR though)

Reply:
Created an issue for it #1469